packages/node-cli/src/metadata/ObjectTypesMetadata.js

```diff
@@ -501,6 +510,16 @@ module.exports = [
 			relatedFiles: [{ type: 'blankscript' }],
 		},
 	},
+	{
+		name: 'Secret',
+		value: {
+			name: 'Secret',
+			type: 'secret',
+			prefix: 'custsecret',
```

Comment:
Probably an underscore is missing at the end of the prefix

Reply:
As the way it is defined in metadata the prefix is `custsecret` without underscore.

Reply:
Fixed. It was already working but for the sake of consistency I added the underslash